src/bin/Makefile

```diff
@@ -7,7 +7,7 @@ BINFILES = align-equal align-equal-compiled acc-tree-stats \
         show-alignments compile-questions cluster-phones \
         compute-wer compute-wer-bootci make-h-transducer \
```

Comment:
Kirill, can you please remove the --natural-base option and hardcode the
default to true (including in the code)?  There was never a need for this.
Dan



On Tue, Mar 22, 2016 at 1:10 PM, Xiaohui Zhang <notifications@github.com>
wrote:

> In src/bin/Makefile
> <https://github.com/kaldi-asr/kaldi/pull/579#discussion_r57027173>:
>
> > @@ -7,7 +7,7 @@ BINFILES = align-equal align-equal-compiled acc-tree-stats \
> >          show-alignments compile-questions cluster-phones \
> >          compute-wer compute-wer-bootci make-h-transducer \
>
> Thanks Krill! sorry for the delay. @danpovey <https://github.com/danpovey>
> , here is a set of comparison of outputs
> /export/a11/xzhang/babel/104-pashto-llp/data/lang/{G.fst, G.bk.fst} G.fst
> is the one produced by this MR. The weights differ a bit, only in the last
> few digits though. Here is the compiled binary:
> /export/a11/xzhang/kaldi/src/lmbin/arpa2fst
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/579/files/ee02fd543e4531aa003cbf296fd6ba50f7d7ea12#r57027173>
>